### ISSUE TITLE
Change callback URL

### ISSUE THREAD

**Issue by dustinbrink** [auth0/express-openid-connect#5]
Would like the ability to specify my own callback url and not just have it default to "/callback"

**Comment by dustinbrink** [comment]
Thank you for adding it!

**Comment by ecs-hk** [comment]
@jfromaniello Many thanks for adding this feature. It is useful in my environment too.

I would like to briefly re-open this issue just to add a comment. The change to support `redirectUriPath` has not been pushed to the npm registry yet.

So for those who would like to install it now, as is (with the working code to support `redirectUriPath`), they can do so by configuring their `package.json` similar to this:

```json
  "dependencies": {
    "body-parser": "^1.19.0",
    "connect-sqlite3": "^0.9.11",
    "ejs": "^2.6.1",
    "eslint": "^5.16.0",
    "eslint-config-strongloop": "^2.1.0",
    "express": "^4.17.1",
    "express-openid-connect": "auth0/express-openid-connect",                       
    "express-session": "^1.16.2",
    "morgan": "^1.9.1"
  }
```

In other words, by using the `"express-openid-connect": "auth0/express-openid-connect"` notation the latest commits from this GitHub repo (rather than from the npm registry) will be installed.

**Comment by JadenSWang** [comment]
was this added to the types file?
